### ISSUE TITLE
[Bug]: Monologue Augmentations - Subsequent actions are formatted incorrectly

### ISSUE THREAD

**Issue by lilyydu** [microsoft/teams-ai#1434]
### Language

Python

### Version

latest

### Description

Subsequent actions performed for monologue augmentations is being formatted incorrectly for "content" object.

For instance, with ListBot:

Creating a list of ingredients is successful-

PromptResponse.message.content is `'{"thoughts":{"thought":"The user wants to create a list of ingredients.","reasoning":"Based on the user\'s request to make a list of ingredients.","plan":"- Create a new list for ingredients."},"action":{"name":"createList","parameters":{"list":"ingredients"}}}'`

However, formatting is incorrect for subsequent actions (e.g., adding ingredients) -

PromptResponse.message.content is `'action:\n  name: addItems\n  parameters:\n    list: ingredients\n    items:\n      - tomatoes\nthoughts:\n  plan: \'- Add the item "tomatoes" to the ingredients list.\'\n  reasoning: User requested to add tomatoes to the list of ingredients.\n  thought: The item "tomatoes" has been added to the ingredients list.'`

This results in JSON parsing errors:
` Validation(valid=False, feedback='No valid JSON objects were found in the response. Return a \n        valid JSON object with your thoughts and the next \n        action to perform.', value=None)`

Or sometimes, a required property is missing. (e.g., "thoughts" is missing here)
`Message(role='user', content="The JSON returned had errors. Apply these fixes:\n'plan' is a required property", function_call=None, name=None), Message(role='assistant', content="action:\n  name: SAY\n  parameters:\n    text: What ingredients would you like to add to the list?\nthoughts:\n  plan: '- Ask the user for the ingredients to add to the list'\n  reasoning: I need to gather the specific items the user wants to include in the\n    ingredients list.\n  thought: Moving forward, I should ask the user for the ingredients to add to the\n    list.\n", function_call=None, name=None)`

### Reproduction Steps

```bash
1. Create a bot with monologue augmentations, register various actions.
2. Perform an action. It should work successfully.
3. Perform other actions, a validation error should occur.
...
```

**Comment by lilyydu** [comment]
Currently in the midst of figuring out where the parsing issue is originating.

**Comment by corinagum** [comment]
Resolved by #1456
